src/client/pages/OfferNew/Introduction/Sidebar/StickyBottomSidebar.tsx

```diff
@@ -76,7 +78,11 @@ export const StickyBottomSidebar: React.FC<Hidable & {
   return (
     <Wrapper isVisible={reallyIsVisible} displayNone={displayNone}>
       <CtaWrapper>
-        {isConnectPaymentAtSignEnabled ? (
+        {isLoadingQuoteCart ? (
+          <Button size="sm" fullWidth disabled style={{ height: '2.75rem' }}>
+            <LoadingDots color={colorsV3.gray500} />
+          </Button>
+        ) : isConnectPaymentAtSignEnabled ? (
```

Comment:
I think we should skip this, it's just a link to open the checkout, right?

Possibly mark the button disabled but no need to show multiple loading state, confused the user